### PR TITLE
Allow duplicate kvk numbers for organizations on debug mode

### PR DESCRIPTION
**Pull request by aghimpu**
https://github.com/teamforus/forus/issues/1177

### CODE REVIEW

**Comment by maxvisser** [comment]
Ah now I see what you meant with your question yesterday; I am sorry I misunderstood. It should use KVK_API_DEBUG instead of app.debug

File: app/Http/Requests/Api/Platform/Organizations/StoreOrganizationRequest.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ public function rules()
             'kvk'                   => [
                 'required',
                 'digits:8',
-                !config('app.debug') ? 'unique:organizations,kvk' : null,
+                !env("KVK_API_DEBUG", false) ? 'unique:organizations,kvk' : null,
```
Comment (maxvisser) [inline app/Http/Requests/Api/Platform/Organizations/StoreOrganizationRequest.php]: you are still inverting it right?

so false becomes true

Comment (aghimpu) [reply app/Http/Requests/Api/Platform/Organizations/StoreOrganizationRequest.php]: Ya, the unique condition is applied only when KVK_API_DEBUG isnt set to true. Second parameter sent to env function is just to consider it false if KVK_API_DEBUG isnt set at all in env.

Comment (aghimpu) [reply app/Http/Requests/Api/Platform/Organizations/StoreOrganizationRequest.php]: So if its not set or set to false then apply the validation rule for unique kvk.

**Pull request merged**